src/bench/bench.cpp

```diff
@@ -126,6 +140,7 @@ void benchmark::BenchRunner::RunAll(Printer& printer, uint64_t num_evals, double
         }
 
         if (!std::regex_match(p.first, baseMatch, reFilter)) {
+             g_testing_setup = nullptr;
```

Comment:
nit: Useless space

Reply:
That's from https://github.com/bitcoin/bitcoin/pull/18013

Reply:
Rebased it out